packages/components/constraints/package.json

```diff
@@ -29,13 +29,12 @@
     "@commercetools-uikit/utils": "10.42.2",
     "@emotion/react": "^11.1.1",
     "@emotion/styled": "^11.0.0",
-    "prop-types": "15.7.2",
```

Comment:
Actually thinking about that, I think we should keep it.
The reason is because the bundles do contain prop types (generated), and the dependency is thus required.

Maybe in the future we can make it an optional peer dependency.

Reply:
If it was an optional peer dependency and somebody wouldn't install it then the bundles would be useless in dev mode? 🤔

Reply:
Good question. Anyway, to avoid issues let's keep it.